packages/synapse-react-client/.storybook/preview.jsx

```diff
@@ -64,12 +64,13 @@ export const globalTypes = {
     title: 'Stack Changer',
     description:
       'Choose the stack that Synapse should point to. You may need to re-authenticate after changing stacks.',
-    defaultValue: 'production',
+    defaultValue: null,
     toolbar: {
       icon: 'database',
       dynamicTitle: true,
       showName: true,
       items: [
+        { value: null, title: 'default (usually production)' },
```

Comment:
These parameters control the "toolbar" control in Storybook. Adding a default `null` value here so we can customize the fallback per-story. Now, we can set certain stories to show mocked data by default in both a regular browser and Chromatic.